app/lib/constants/settings/general.rb

```diff
@@ -101,6 +101,32 @@ module General
           description: "Determines which tags are shown on the homepage righthand sidebar",
           placeholder: "List of valid, comma-separated tags e.g. help,discuss,explainlikeimfive,meta"
         },
+        smtp_settings: {
+          address: {
+            description: "Address of the remote mail server",
+            placeholder: ""
+          },
+          port: {
+            description: "",
+            placeholder: ""
+          },
+          authentication: {
+            description: "",
+            placeholder: "plain"
+          },
+          user_name: {
+            description: "if your mail server requires authentication, set the username in this setting",
+            placeholder: ""
+          },
+          password: {
+            description: "If your mail server requires authentication, set the password in this setting.",
```

Comment:
I feel like `in this setting` is redundant. I tried to make it a bit more explicit but feel free to ignore if you don't agree with the suggestion. 
```suggestion
            description: "If your mail server requires authentication, copy the password from your server.",
```